### PR TITLE
init

### PR DESCRIPTION
**Pull request by rajatagarwal-ibm**
SKIP UPGRADE TEST: init

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: .catalog-onboard-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
+    offering_id: c0e62838-544b-4d02-a946-e48cd5dd6fa0
+    variations:
+      - name: banking
+        mark_ready: true
+        install_type: fullstack
+        scc:
+          instance_id: 1c7d5f78-9262-44c3-b779-b28fe4d88c37
+          region: us-south
+          scc:
```
Comment (ocofaigh) [inline .catalog-onboard-pipeline.yaml:15]: you have scc section duplicated here

File: .catalog-onboard-pipeline.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+offerings:
+  - name: customer-care-gen-ai-app-config-da
+    kind: solution
+    catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
+    offering_id: c0e62838-544b-4d02-a946-e48cd5dd6fa0
```
Comment (ocofaigh) [inline .catalog-onboard-pipeline.yaml:7]: what is this offering ID? I dont see any entry in our catalog for it?

**Review by vburckhardt — APPROVED** [review]

**Pull request merged**